Makefile

```diff
@@ -1,428 +0,0 @@
-# Select architecture: RV32 or RV64.
```

Comment:
Could keep the makefile for the targets that aren't supported yet and just defer to CMake by default?
e.g.
```
csim: $(BUILD_DIR)
    cmake -S . -B $(BUILD_DIR) -GNinja && ninja -C $(BUILD_DIR)
```

Or to avoid needing ninja:
```
csim: $(BUILD_DIR)
    cmake -S . -B $(BUILD_DIR) && $(MAKE) -C $(BUILD_DIR)
```